### DISCUSSION TITLE
No Progress bar showing in Slurm output file

### DISCUSSION THREAD

**Discussion by Cing2** [Lightning-AI/pytorch-lightning#19227] [DDP / multi-GPU / multi-node] [answered]
The title explains exactly the problem I have. 
I would like to see the TQDM progress bar updates in the slurm output log. 
It works on my computer and I get the progress bar in my terminal. 

However, when I run a sbatch script of the model I do not see the progress bar updates in the slurm output file or somewhere else in the logs directory.  I tried both the tqdm and rich progress bar but with both I do not see any output.

Is this the intended behaviour for running in a slurm system? 
If so it there a way to get tqdm to output to the slurm output file or maybe a way to redirect it to another file.

I made sure I am using the latest Ligthning and tqdm version.  I also search everywhere on the lightning docs and github for a similar question but i could not find any. 

Any help or solutions are appreciated. 
Thank you.

**Comment by Cing2** [comment, marked as answer]
Okay, I found my problem. It was a mistake from my part, I am sorry for the issue.
I was using the pytorch ligthning + hydra template to construct my code and I used hydra to change the progress bar which was default set to rich to tqdm, however, I did this wrong and thus I was still using the rich progress bar on the server. This one does not provide intermediate step updates.

**Reply by snknitin** [reply]
Hi, Could you elaborate on what this change was ? I'm using the same template and getting issues with progressBar visibility. It doesn't even write the epochs to the log. and not visible in terminal either.
